client/src/assets/styles/style_props.ts

```diff
@@ -0,0 +1,65 @@
+// Styles for resources page.
+
+interface Style {
```

Comment:
I don't want to block the CL on this bc I know the pattern was introduced before but I'm worried about defining style props. What I have *started* doing is a map of styling a parent can pass in. The style requirements for each component is going to vary wildly so I don't think defining an interface makes sense here 

https://github.com/BlueConduit/open-data-platform/blob/1307a59ec063fb0559896d59290e8779e6d6ebda/client/src/components/MapLegend.vue#L80

Reply:
I can change to this but I feel like the complexity of this surpasses the value we get from a shared section component, which really only saves a few lines in html from each component. Let's discuss in standup.

Reply:
Removed use of the component so this no longer applies.